src/commands/image/imageSource/buildImageInAzure/BuildImageStep.ts

```diff
@@ -63,11 +72,26 @@ export class BuildImageStep extends ExecuteActivityOutputStepBase<BuildImageInAz
     }
 
     protected createFailOutput(context: BuildImageInAzureImageSourceContext): ExecuteActivityOutput {
+        let loadMoreChildrenImpl: (() => Promise<AzExtTreeItem[]>) | undefined;
+        if (this.acrBuildError) {
+            loadMoreChildrenImpl = () => {
+                const buildImageLogsItem = new GenericTreeItem(undefined, {
+                    contextValue: createActivityChildContext(['logsLinkItem']),
+                    label: localize('buildImageLogs', 'Click to view build image logs'),
+                    iconPath: new ThemeIcon('note', new ThemeColor('terminal.ansiWhite')),
```

Comment:
nit: What about using link-external? Kind of makes sense to me since it's opening a new file. Or maybe file-symlink-file?

Icons here: https://microsoft.github.io/vscode-codicons/dist/codicon.html

Reply:
- Link-external: 
<img width="268" alt="image" src="https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/c715a5b8-4baf-4869-a1fb-5420f717a9cf">

- File-symlink-file:
<img width="345" alt="image" src="https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/9baed23c-6bc3-476d-8773-03646f81006d">